app/assets/javascripts/views/restrooms/search.coffee

```diff
@@ -62,9 +62,13 @@ class Refuge.Restrooms.Search
 
 
   _searchCurrentLocation: =>
-    @_geocoder.getCurrentLocation().then (currentCoords) =>
-      @_updateForm(currentCoords.lat, currentCoords.long, "Current Location")
-      @_form.submit()
+    @_geocoder.getCurrentLocation()
+    .then (currentCoords) =>
```

Comment:
Ah if the rest of the code base doesn't indent the `.then` then you should leave it, but the next line below the `.then`s are now double indented :P My personal preference is to indent the `.then` so it's more obvious that it's chaining off the line above it.